ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -1145,6 +1145,74 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-usertags-custom-security-groups
+- as: aws-ipi-wavelength-byo-subnet-ovn-day2-f14
+  cron: 0 1 11,25 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@amd64 and @aws-ipi and @network-ovnkubernetes and not @fips'
+      EDGE_ZONE_TYPE: wavelength-zone
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-aws-ipi-localzone-byo-subnet-ovn-day2
```

Comment:
@yunjiang29 Seen from the overall design, looks like we categorize the EDGE_ZONE_TYPE `wavelength-zone` and `local-zone`(default) as the two types of the edge zone testing, so do we want to use 
`edgezone` in the workflow name to be more clear?

Reply:
@gpei  Good point, actually I have the same thought as you.
In the initial version of this PR, I tried to replace `localzone` with `edge_zone`, but it does take significant effort to consolidate local-zone, wavelength-zone and edge-zone in the repo, especially some of developer's steps are involved, so in this PR I choose an easy way to do this: _wavelength-zone is a kind of local-zone_, every _localzone_ in the script, also means it's possible a _wavelength-zone_

But I'm happy to consolidate all of them if the current changes are confusing. (a separate PR would be better)

Reply:
Got it, It just doesn't look as clear as the logic in the flexy-template, it's ok to me to put wavelength-zone as a kind of local-zone in the script.

Reply:
>  looks like we categorize the EDGE_ZONE_TYPE wavelength-zone and local-zone(default) as the two types of the edge zone testing, so do we want to use
edgezone in the workflow name to be more clear?

@gpei @yunjiang29 good point! ACK with Yunfei, everything was created to Local Zone, now we have WL that will fit it.
One of my goals* is to consolidate everything referencing "local*zone" to "edge-zone", including the ones using "wavelength-zone" (I would also consolidate jobs, a single job should test both in a single execution to save costs keeping covering the features which are very similar). If we'll not change here I will be happy to receive your feedback in the PR I will open to do it.

*https://github.com/openshift/enhancements/pull/1510/files#diff-37d19e83b868514c474a714f6863c525e9ef5ebd72c8487a6ba708e42b4ec02cR486